tools/run_tests/helper_scripts/run_grpc-node.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
I don't think this is supposed to be deleted

Reply:
You're right. I fixed that.